src/compas/geometry/transformations/quaternions.py

```diff
@@ -0,0 +1,339 @@
+"""
```

Comment:
use the "raw" flag to avoid having to escape backslashes in math formulas: `r"""`